ArjunaCore/arjuna/tests/classes/com/hp/mwtests/ts/arjuna/recovery/RecoveryDriverUnitTest.java

```diff
@@ -46,20 +46,20 @@ public void testInvalid () throws Exception
         try
         {
             rd.asynchronousScan();
-            
-            fail();
+
+            fail("Recovery driver asynchronously calls a non-existing host and port 'foobar:0'. Failure is expected.");
```

Comment:
What does `foobar:0` signify?

Reply:
From my understanding it's a non-existent `hostname:port` sequence. I think it's good point to rename it to be more descriptive.